### ISSUE TITLE
Popup in admin panel on items list blocks `Ctrl+Shift+S` in Edge

### ISSUE THREAD

**Issue by xperiandri** [OrchardCMS/OrchardCore#9719]
### Describe the bug
![image](https://user-images.githubusercontent.com/2365592/122524950-6613bd80-d021-11eb-8629-19eac348aac4.png)

When popup is open
![image](https://user-images.githubusercontent.com/2365592/122524982-6dd36200-d021-11eb-8602-686d7c17c452.png)
![image](https://user-images.githubusercontent.com/2365592/122524988-6f9d2580-d021-11eb-8fa0-6a1e7b46af1e.png)

`Ctrl+Shift+S` shortcut of making pageshots (screenshots) is not invoked

### To Reproduce
1. In Edge go to admin panel, content items list
2. Click on Show or Sort drop-down button
3. Press `Ctrl+Shift+S`
4. Nothing happens
5. Close popup
6. Press `Ctrl+Shift+S`
7. Open a pageshot preview

### Expected behavior
Pageshot preview can be always invoked on the pages

1.0.0-rc-16308

**Comment by agriffard** [comment]
Did you try `Win+Shift+S` ?

**Comment by xperiandri** [comment]
No, Microsot Edge has an internal shortcut `Ctrl+Shift+S` for pageshots

**Comment by sebastienros** [comment]
Issue in the jquery plugin https://github.com/snapappointments/bootstrap-select/issues/2626
